README.md

```diff
@@ -1 +1,84 @@
-# Stock-Market
\ No newline at end of file
+# Stock-Market
+
+A web application that provides you with the data about the most active trading stocks for the day.
+
+## Deployment
+
+https://stocks-market-app.herokuapp.com/
+
+## Major Languages
+
+- HTML
+- CSS
+- JavaScript
+- React JS
+- Redux
+- API
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup
+
+To get a local copy up and running follow these simple steps.
+
+To setup Stock-market project, in the repo page:
+click on code (dropdown list) > Download as ZIP;
+or open terminal of path you want to install project and run this command <br>
+
+```sh
+  git clone git@github.com:MohammedElseddik/Stock-Market.git
+```
+
+## 🛠 Installation & Set Up to run the project locally
+
+1. Install Node Modules
+
+   ```sh
+   npm install
+   ```
+
+2. Start the development server
+
+   ```sh
+   npm start
+   ```
+   
+## 🛠 Run test
+
+  ```sh
+   npm test
+  ```
+
+the project will be opened in [http://localhost:3000] on the browser.
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.